mindmaps-graph/src/main/java/io/mindmaps/example/PokemonGraphFactory.java

```diff
@@ -47,6 +48,26 @@
     private static RelationType evolution;
     private static RelationType hasType;
     private static RelationType superEffective;
+    private static Entity normal;
```

Comment:
Hahahaha I remember doing this when we first took a whack at internalising ids :)